radars/primary/quadrants/open-edx/rings/hold/blips/tech-radar.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Tech Radar",
+    "ring": "Hold",
```

Comment:
I think this should be `Provisional`: https://openedx.atlassian.net/wiki/spaces/AC/pages/2844786770/Open+edX+Technology+Radar#Rings

Reply:
Changed and pushed.